### ISSUE TITLE
ASCII control characters/sequences in WSL Ubuntu

### ISSUE THREAD

**Issue by fatherofinvention** [vit-project/vit#220]
Hello - 

The attached screenshot shows how the [Shift In control character](https://en.wikipedia.org/wiki/Shift_Out_and_Shift_In_characters) is rendering in WSL Ubuntu. The font in use is [Ubuntu Mono](https://fonts.google.com/specimen/Ubuntu+Mono).

`SI (0x0F, ^O) activates the G0 character set;`
[Source](http://man7.org/linux/man-pages/man4/console_codes.4.html)

I am not sure if this is a WSL issue, a code page issue, a font issue, or a Python issue. I am hoping someone more knowledgeable can comment on whether this is a fundamental limitation of WSL's support of control characters or if this can be resolved by changing a setting. I have tried numerous monospace fonts without any luck.

![SI-control-characters](https://user-images.githubusercontent.com/24985048/71548000-7bdc8100-295c-11ea-8ab8-cbd48f394436.jpg)

Thank you.

**Comment by thehunmonkgroup** [comment]
This is most likely a system-level issue, although I can't say for sure. I doubt I'll be able to help much.

I have run into similar issues in the past with font sets not displaying non-character codes correctly, and in those cases, I resolved them by making sure the shell session loading whatever (it was vim in my case) had a proper set of UTF8 locales set, like so:

```
[:~] $  locale
LANG=en_US.UTF-8
LANGUAGE=en_US:en
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC=en_US.UTF-8
LC_TIME=en_US.UTF-8
LC_COLLATE="en_US.UTF-8"
LC_MONETARY=en_US.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=en_US.UTF-8
LC_NAME=en_US.UTF-8
LC_ADDRESS=en_US.UTF-8
LC_TELEPHONE=en_US.UTF-8
LC_MEASUREMENT=en_US.UTF-8
LC_IDENTIFICATION=en_US.UTF-8
LC_ALL=
```

**Comment by fatherofinvention** [comment]
Thank you @thehunmonkgroup ! I appreciate the suggestion, I tried it in this case but it did not work.

I posted this same issue to the microsoft / terminal repo and it seems they are aware and closed it as a dupe. Given that I think we can close it here too.

For reference this is my issue and the one they already had: 

Mine:
https://github.com/microsoft/terminal/issues/4072

Original:
https://github.com/microsoft/terminal/issues/3487
